README.md

```diff
@@ -132,3 +132,48 @@ This will only submit the data and not validate.
 If you are working with large VCF files and find that validation takes a very long time, you can add the
 argument `--shallow` to the command, which will validate only the first 10,000 lines in each VCF. Note that running
 shallow validation will **not** be sufficient for actual submission.
+
+
+## Leveraging Nextflow to parallelize the validation process  
```

Comment:
Do you think this should go in the main readme or in a separate "advanced usage" doc?  It feels technical but on the other hand it's actually pretty important...

Reply:
I hesitated but ultimately felt like it was too short  for its own document. And since it will be used by everyone regardless of if they want to use advance feature I think we'll need to have some of it in the main readme anyway.